examples/README.md

```diff
@@ -811,6 +813,7 @@ spec:
 * `limit` is the maximum number of times the container will be retried.
 * `retryPolicy` specifies if a container will be retried on failure, error, or both. "Always" retries on both errors and failures. Also available: "OnFailure" (default), "OnError"
 * `backoff` is an exponential backoff
+* `nodeAntiAffinity` prevents running steps on the same host.  Current implementation allows only empty `nodeAntiAffinity` (i.e. `nodeAntiAffinity: {}`) and by default it uses label `kubernetes.io/hostname` as the selector.
```

Comment:
is this still correct?

Reply:
yes, it uses the default selector for hostname `kubernetes.io/hostname`. It seems to be a k8s wide standard according to this: https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#kubernetesiohostname